pkg/kubectl/cmd/create.go

```diff
@@ -201,8 +201,10 @@ func NameFromCommandArgs(cmd *cobra.Command, args []string) (string, error) {
 
 // CreateSubcommandOptions is an options struct to support create subcommands
 type CreateSubcommandOptions struct {
-	// Name of resource being created
+	// Name of resource instance being created (ex: my-pod, my-namespace)
 	Name string
+	// Resource being created (ex: pod, namespace)
+	Resource string
```

Comment:
oh you mean, users should be allowed to write `kubectl create namespace.groupname my-namespace`?
You are right. Will fix, thanks.